src/configure/configure.ts

```diff
@@ -648,14 +655,54 @@ class Orchestrator {
             throw error;
         }
     }
+
+    private createProvisioningConfiguration(templateId: string, branch: string, pipeplineTemplateParameters: { [key: string]: string }, mode: provisioningMode): ProvisioningConfiguration {
+        return {
+          id: null,
+          branch,
+          pipelineTemplateId: templateId,
+          pipelineTemplateParameters: pipeplineTemplateParameters,
+          provisioningMode: mode,
+        } as ProvisioningConfiguration;
+      }
+
+    private async configurePipelineRemotely(): Promise<void>{
+        const provisioningConfigurer = new ProvisioningConfigurer(this.localGitRepoHelper);
+        const template = this.inputs.pipelineConfiguration.template as RemotePipelineTemplate;
+        await provisioningConfigurer.CreatePreRequisiteParams(this.inputs);
+        const provisioningConfiguration =  this.createProvisioningConfiguration(template.id, this.inputs.sourceRepository.branch, this.inputs.pipelineConfiguration.params , provisioningMode.draft );
+        try {
+            // Initially send the provisioning request in draft mode to get workflow files
+            const provisioningServiceDraftModeResponse  = await provisioningConfigurer.createProvisioningPipeline(provisioningConfiguration, this.inputs);
+            if (provisioningServiceDraftModeResponse.id  != "" ) {
+                const OrgAndRepoDetails =  this.inputs.sourceRepository.repositoryId.split('/');
+
+                // Monitor the provisioning pipeline
+                const provisioningDraftPipeline =  await provisioningConfigurer.checkProvisioningPipeline(provisioningServiceDraftModeResponse.id, OrgAndRepoDetails[0], OrgAndRepoDetails[1], this.inputs );
+
+                provisioningConfiguration.provisioningMode =  provisioningMode.complete;
```

Comment:
where are we showing draft files to user and notification to commit & push

Reply:
In postStep function, we are showing draft files and notifying user for to modify and commit
https://github.com/microsoft/vscode-deploy-azure/pull/154/files#diff-df72344d1fd58d154bf36fc7942641e5R88